chall02/cvinolo-.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+# **************************************************************************** #
+#                                                                              #
+#                                                         :::      ::::::::    #
+#    cvinolo-.py                                        :+:      :+:    :+:    #
+#                                                     +:+ +:+         +:+      #
+#    By: cvinolo- <cvinolo-@student.42.fr>          +#+  +:+       +#+         #
+#                                                 +#+#+#+#+#+   +#+            #
+#    Created: 2020/04/06 18:54:53 by cvinolo-          #+#    #+#              #
+#    Updated: 2020/04/06 18:54:53 by cvinolo-         ###   ########.fr        #
+#                                                                              #
+# **************************************************************************** #
+
+#
+# Constelación Serpens (Chall02) / String to Morse
+#
+#Implemented shebang to indicate (in unix system)
+#where the environment variable "python3" is.
+#
+#Solved with a dictionary to list morse alphabet with a
+#key, value pair based on modern english alphabet
+#(including space character for a more friendly implementation).
+#
+#Defined functions to obtain the command usage and the
+#morse message, in order to separate the logic from the visual.
+#
+#Implemented try/except to handling all possibles errors and
+#exceptions we don't want in our program.
+#
+
+import sys
+
+morse_alphabet = {
```

Comment:
If you use global it's better to write them in upper case

Reply:
I'd like to know why it's better to write them in upper case. In [solution.py](https://github.com/42madrid/remote-challs/blob/master/chall02/solution.py#L5) you can see on line `5` a global variable in lower case. Also, [doing a search](https://www.google.com/search?sxsrf=ALeKk03YnGGBvNYqptHJ88yEYEW_X7NAcg%3A1586377506435&ei=IjOOXpWFGpLMgweIrYzADQ&q=python+naming+convention+global+variable&oq=Python+naming+convention+globa&gs_lcp=CgZwc3ktYWIQARgAMgUIABDLATIICAAQCBANEB4yCAgAEAgQDRAeOgQIABBHOgQIIxAnOgQIABBDOgUIABCDAToHCAAQFBCHAjoCCAA6BggAEBYQHkosCBcSKDBnMTAyZzk0Zzk5ZzEwMWc5MWc5MWc4N2cxMDBnODlnOTBnOTVnNjhKHggYEhowZzFnMWcxZzFnMWcxZzFnMWcxZzVnNWcyMFCH-RFYkNISYJHaEmgAcAh4AIABggGIAZsVkgEEMjUuNpgBAKABAqABAaoBB2d3cy13aXo&sclient=psy-ab) you're always going to find something related to:
- **Global variables** should be all lowercase.
- Words in a **global variable name** should be separated by an underscore.

Reply:
According to PEP8s rule on constants:

`Constants are usually defined on a module level and written in all capital letters with underscores separating words. Examples include MAX_OVERFLOW and TOTAL.`

In this case, `morse_alphabet` is used as constant (constant keyword doesn't exist in Python but the concept does). Style guides are really subjective, but PEP8 is the most used and respected one.

As for `solution.py`, you will have to ask pistache 😅. Just kidding, we should have said that "we recommend" writing them in upper case, not that "it's better" to write them in upper case.